accessible/field-segment.component.js

```diff
@@ -144,7 +150,12 @@ blocklyApp.FieldSegmentComponent = ng.core.Component({
   // Confirm a selection for dropdown fields.
   selectOption: function() {
     if (this.optionValue == Blockly.Msg.RENAME_VARIABLE) {
```

Comment:
String comparisons for option selections make me sad. Please file an issue to make this compare the index instead of the strings.